release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -51,6 +51,11 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-9-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-9-install-ign-removed"]
+==== Installation Ignition config is removed upon boot
+
+Nodes installed with the `coreos-installer` program previously retained the installation Ignition config in the `/boot/ignition/config.ign` file. Starting with the {product-title} 4.9 installation image, that file is removed when the node is provisioned. This change does not affect clusters that were installed on previous {product-title} versions because they are still using an older bootimage.
```

Comment:
Maybe

```suggestion
Nodes installed with the `coreos-installer` program previously retained the installation Ignition config in the `/boot/ignition/config.ign` file. Starting with the {product-title} 4.9 installation image, that file is removed when the node is provisioned. This change does not affect clusters that were installed on previous {product-title} versions because they still use an older boot image.
```

Reply:
Went with your suggestion, thanks!